parsers/jscript.c

```diff
@@ -1658,27 +1652,27 @@ static bool parseMethods (tokenInfo *const token, const tokenInfo *const class,
 		else if (isType (token, TOKEN_KEYWORD) &&
 				 (isKeyword (token, KEYWORD_get) || isKeyword (token, KEYWORD_set)))
 		{
-			tokenInfo *savedToken = newToken ();
-			copyToken (savedToken, token, true);
+			tokenInfo *saved_token = newToken ();
```

Comment:
Could you make another commit for changing the variable name?

BTW, we don't define a convention about the names having an in-file scope.
This change, /saveToken/saved_token/ is acceptable.